var/spack/repos/builtin/packages/r-caroline/package.py

```diff
@@ -10,10 +10,8 @@ class RCaroline(RPackage):
     """caroline: A Collection of Database, Data Structure, Visualization,
        andUtility Functions for R"""
 
-    homepage = "https://cran.r-project.org/package=caroline"
-    url      = "https://cran.r-project.org/src/contrib/caroline_0.7.6.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/caroline"
+    homepage = "https://cloud.r-project.org/package=caroline"
+    url      = "https://cloud.r-project.org/src/contrib/caroline_0.7.6.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/caroline"
 
     version('0.7.6', sha256='e7ba948f7d87f091b498dd0eec2ca4fdad7af4e2bbb67e0945c2f0d3f2eadda9')
-
-    depends_on('r@1.8.0:', type=('build', 'run'))
```

Comment:
According to https://cloud.r-project.org/package=caroline it requires R (≥ 1.8.0)

Reply:
I think I had taken those out some time back and forgot about them. I will make another pass through these and catch those although I really can not see anyone using versions of R that old.

Reply:
@adamjstewart Also, there are some packages that did not have the R version specified even for newer versions of R. It looks like I missed some of those. Not sure if I will have another commit this evening but let me get those in before you spend too much time on that aspect.

Reply:
@adamjstewart Sorry this took so long but I had trouble finding blocks of time to work on it. The packages have R versions specified now.